### ISSUE TITLE
doc generation is broken because of extra help param --ids for vmss

### ISSUE THREAD

**Issue by erich-wang** [Azure/azure-cli#821]
Doc build process encountered the following error when running `make xml`

> Exception occurred:
>   File "d:\msdn\azure\my-azure-cli\src\azure\cli\_help.py", line 311, in _load_from_data
>     raise HelpAuthoringException('Extra help param {0}'.format(extra_param['name']))
> azure.cli._help.HelpAuthoringException: Extra help param --ids

It seems the error is caused by the following code in /src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```
helps['vmss get-instance-view'] = """
    type: command
    parameters: 
        - name: --ids
          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
"""

helps['vmss reimage'] = """
    type: command
    parameters: 
        - name: --ids
          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
"""

helps['vmss show'] = """
    type: command
    parameters: 
        - name: --ids
          short-summary: "One or more scale set or specific VM instance IDs. If provided, no other 'Resource Id' arguments should be specified."
"""
```

**Comment by tjprescott** [comment]
The ids parameter is in fact part of the argument for this command. The fact that it was previously omitted was in fact a bug. @BurtBiel it seems to be complaining at line 311 of `_help.py`. However, this does not happen when you run a command (`az vmss show -h`) or during the tests. It might be something in `azhelpgen.py` where ids is not attached in the command table and thus ends up being seen as extra?

**Comment by BurtBiel** [comment]
Fixed by #832
